aviatrix/resource_aviatrix_transit_gateway_peering.go

```diff
@@ -93,32 +93,25 @@ func resourceAviatrixTransitGatewayPeering() *schema.Resource {
 				Description: "(Optional) Enable peering over private network. Insane mode is required on both transit gateways. Available as of provider version R2.17.1",
 			},
 			"enable_single_tunnel_mode": {
-				Type:     schema.TypeBool,
-				Optional: true,
-				Default:  false,
-				ForceNew: true,
-				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
-					return !d.Get("enable_peering_over_private_network").(bool)
-				},
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				ForceNew:    true,
 				Description: "Enable peering with Single-Tunnel mode.",
 			},
 			"enable_insane_mode_encryption_over_internet": {
-				Type:          schema.TypeBool,
-				Optional:      true,
-				Default:       false,
-				ForceNew:      true,
-				ConflictsWith: []string{"enable_peering_over_private_network", "enable_single_tunnel_mode"},
-				RequiredWith:  []string{"tunnel_count"},
-				Description:   "Enable Insane Mode Encryption over Internet. Type: Boolean. Default: false.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				ForceNew:    true,
+				Description: "Enable Insane Mode Encryption over Internet. Type: Boolean. Default: false.",
 			},
 			"tunnel_count": {
-				Type:          schema.TypeInt,
-				Optional:      true,
-				ForceNew:      true,
-				ValidateFunc:  validation.IntBetween(2, 20),
-				ConflictsWith: []string{"enable_peering_over_private_network", "enable_single_tunnel_mode"},
-				RequiredWith:  []string{"enable_insane_mode_encryption_over_internet"},
-				Description:   "Number of public tunnels. Only valid with 'enable_insane_mode_encryption_over_internet'. Type: Integer. Default: 4. Valid Range: 2-20.",
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.IntBetween(2, 20),
+				Description:  "Number of public tunnels. Only valid with 'enable_insane_mode_encryption_over_internet'. Type: Integer. Default: 4. Valid Range: 2-20.",
```

Comment:
Description says default 4, but we don't set a default value because it is optional. Maybe change the wording to state controller defaults to 4.

Reply:
Removed it.